api-test/src/main/resources/idRepository/AddIdentityV2/AddIdentityV2.yml

```
@@ -0,0 +1,643 @@
+  
+  "version": "v2",
+  "introducerRID": "212124324784879",
+  "introducerUIN": "212124324784879",
+   
+   "category": "individualBiometrics",
+   "requesttime": "$TIMESTAMP$"
+}'
+      output: '{
+  "status":"ACTIVATED"
+}'
+  IdRepository_AddIdentityV2_WithEmptyVerifiedAttributes_Smoke_Pos:
```

YAMLlint (1.38.0)

[error] duplication of key "IdRepository_AddIdentityV2_WithEmptyVerifiedAttributes_Smoke_Pos" in mapping

(key-duplicates)